include/zephyr/net/prometheus/pm_collector.h

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
Minor nit, but the commit message does not follow English grammar and looks bad. Could you fix that by starting sentences with capital letter etc.